.github/workflows/ci.yml

```
@@ -1,8 +1,8 @@
 on: 
     pull_request:
-      branches: [ master ]
+      branches: [ main ]
```

Run on both master and main for transition:

```suggestion
      branches:
      - main
      - master
```